docs/src/main/paradox/scala/http/common/de-coding.md

```diff
@@ -3,9 +3,9 @@
 The [HTTP spec](http://tools.ietf.org/html/rfc7231#section-3.1.2.1) defines a `Content-Encoding` header, which signifies whether the entity body of an HTTP message is
 "encoded" and, if so, by which algorithm. The only commonly used content encodings are compression algorithms.
 
-Currently Akka HTTP supports the compression and decompression of HTTP requests and responses with the `gzip` or
+Currently, Akka HTTP supports the compression and decompression of HTTP requests and responses with the `gzip` or
 `deflate` encodings.
-The core logic for this lives in the @scaladoc[akka.http.scaladsl.coding](akka.http.scaladsl.coding.index) package.
+The core logic for this, lives in the @scala[@scaladoc[akka.http.scaladsl.coding](akka.http.scaladsl.coding.index) package.]@java[@javadoc[akka.http.javadsl.coding.Coder](akka.http.javadsl.coding.Coder) enum class.]
```

Comment:
Not sure about this comma. Reduces to "it, lives in ..." which looks very poetic.